AndroidNativeKotlinTemplate/app/src/main/AndroidManifest.xml

```diff
@@ -20,6 +20,16 @@
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
+
+            <!-- Uncomment when enabling log in via Salesforce UI Bridge API generated QR codes -->
+            <!-- This declares the app's support for QR code login deep links.  The scheme can be customized by the app to match any scheme used in the QR Code's URL -->
+<!--            <intent-filter>-->
```

Comment:
Note this intent filter is attached to the app's main activity now, which was the crux of fixing this issue.